### PR TITLE
Forward Qt info/debug console (as well as qml console.log) messages to android's logcat

### PR DESCRIPTION
**Pull request by nirvn**
This greatly, greatly helps debugging stuff on Android via adb.

### CODE REVIEW

File: src/app/main.cpp [hunk 1]
```diff
@@ -49,6 +91,8 @@ int main( int argc, char **argv )
 
 #ifdef ANDROID
   QgsApplication app( argc, argv, true, AndroidPlatformUtilities().packagePath() + QStringLiteral( "/resources" ) );
+  qInstallMessageHandler(qfMessageHandler);
```
Comment (m-kuhn) [inline src/app/main.cpp:94]:
```suggestion
#ifdef ANDROID
  qInstallMessageHandler(qfMessageHandler);
#endif
```

**Comment by m-kuhn** [comment]
Nice one !

Could you run clang-format (or scripts/prepare-commit / astyle) on it?

File: src/app/main.cpp (hunk 1 above)
Comment (nirvn) [reply src/app/main.cpp:94]: Why adding the ifdef here, I'm already in a ifdef ANDROID (check line 92)

**Comment by nirvn** [comment]
@m-kuhn , script executed, C++ code styled, cheers.

Let's merge this! :)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-978-e5dcbc8cc83fd7bf70c76f34e1b36794af079f8a-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-978-e5dcbc8cc83fd7bf70c76f34e1b36794af079f8a-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-978-e5dcbc8cc83fd7bf70c76f34e1b36794af079f8a-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-978-e5dcbc8cc83fd7bf70c76f34e1b36794af079f8a-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-978-2d573fe7995fa9ad060df73b416d891310f03adb-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-978-2d573fe7995fa9ad060df73b416d891310f03adb-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-978-2d573fe7995fa9ad060df73b416d891310f03adb-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-978-2d573fe7995fa9ad060df73b416d891310f03adb-x86_64.apk)

**Pull request merged**